evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
     </div>
 {% endblock %}
 
+{% block modals %}
+{{ block.super }}
+    {% trans 'Import participants' as title %}
+    {% blocktrans asvar question %}Do you really want to upload the Excel file with <strong id="data-label"></strong>? {% endblocktrans %}
```

Comment:
"upload" is a bit inaccurate here. In the import run, nothing is uploaded anymore.